tslint.json

```diff
@@ -16,7 +16,15 @@
         "variable-name": false,
         "forin": false,
         "member-ordering": false,
-        "only-arrow-functions": false
+        "only-arrow-functions": false,
+        "trailing-comma": [
+            true,
+            {
+                "multiline": {
+                    "functions": "never"
```

Comment:
I think we can keep it `always` for functions as well. The reason it's not set in eslint is due to js not supporting it.